rewards.staging.mainnet.env

```diff
@@ -0,0 +1,7 @@
+# config
+export RPC_URL=https://eth.llamarpc.com
+export CHAIN=mainnet
+export FACTORY=0x290CAB97a312164Ccf095d75D6175dF1C4A0a25F # see symbiotic docs
+
+# artifacts
+export STAKER_REWARDS=0x7Cbc03F716B372116d6237F1B411f7506c279477
```

Comment:
We are deprecating the old factory, and the UI + backend will support only the new one for now, hence it is needed to redeploy (only the events changed in the new version)